### DISCUSSION TITLE
FFMpeg installed... now where is it?

### DISCUSSION THREAD

**Discussion by tritrek** [Homebrew/discussions#4436] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.14
ORIGIN: https://github.com/Homebrew/brew
HEAD: ac0663ae5d2d4c76bb62363592d77dbad49ac30c
Last commit: 3 days ago
Core tap JSON: 19 Apr 08:10 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.87.0 => /usr/bin/curl
macOS: 13.3.1-arm64
CLT: 14.3.0.0.1.1679647830
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Hey, I have installed FFMpeg with "brew install ffmpeg" and made sure it's linked, too. Now, I'm trying to locate the installed libraries from Audacity, but I don't seem to find them anywhere. When I search 'This Mac' for ffmpeg it finds nothing...
Any help is appreciated ;)
Thanks

**Comment by SMillerDev** [comment, marked as answer]
`/opt/homebrew/opt/ffmpeg`

**Comment by tritrek** [comment]
Thanks, but Audacity is just not taking it. I can see that file there but Audacity doesn't accept it. I guess I'll need to open a discussion with the Audacity guys...

**Comment by tritrek** [comment]
OK, there's probably some kind of disconnect between Brew <> FFMpeg <> Audacity, becuase Audacity only accepts the older (5.0) version, not the new 6.0.
As per: https://forum.audacityteam.org/t/macos-ffmpeg-library-not-found/73445
Tested: yup, that's it.

**Reply by alebcay** [reply]
In that case, Homebrew also has FFmpeg 5.x available, `brew install ffmpeg@5`.
